<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleWidget</class>
 <widget class="QWidget" name="SaleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>销售信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/main_logo.png</normaloff>:/new/prefix1/images/main_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#SaleWidget{background-color: rgb(0, 255, 255);}</string>
  </property>
  <widget class="QTableWidget" name="saletableWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>160</y>
     <width>611</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>订单号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>折扣率</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>订单日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>联系电话</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>订单总利润（元）</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>修改订单</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>办理退货</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询订单号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>111</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>办理售货</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>350</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>总利润：</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
